openrtb_ext/imp_kobler.go

```diff
@@ -0,0 +1,5 @@
+package openrtb_ext
+
+type ExtImpKobler struct {
+	Test *bool `json:"test"`
```

Comment:
No need to declare it as a pointer. Dafault value for bool is false. 
Just assign it without nil check: `testMode = impExt.Test`
Please keep in mind that if a request has more than one imp then `testMode` will have a value from the last impression.

Reply:
I also check `i == 0`, so it should only be for the first imp we do this parsing.